storage/storage.go

```diff
@@ -90,55 +90,132 @@ func (storage *Storage) getAllChunks(context context.Context, buildId string) ([
 	return buildChunks, nil
 }
 
-func (storage *Storage) GetTestLogLines(context context.Context, buildId string, testId string) (LogIterator, error) {
+func (storage *Storage) getBuildAndTestChunks(context context.Context, buildId string) ([]LogChunkInfo, []LogChunkInfo, error) {
 	chunks, err := storage.getAllChunks(context, buildId)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
+	}
+
+	buildChunks := []LogChunkInfo{}
+	for i := 0; i < len(chunks); i++ {
+		if chunks[i].TestID == "" {
+			buildChunks = append(buildChunks, chunks[i])
+		}
 	}
 
 	testChunks := []LogChunkInfo{}
 	for i := 0; i < len(chunks); i++ {
 		// Find our test id
```

Comment:
this is no longer the case

Reply:
Sorry, what is?

Reply:
sorry: I meant that we're no longer finding _our_ test id, but rather any chunk with a test id

Reply:
Oh the comment...

Reply:
yes. If we prefer I can omit pedantic things from my reviews.

Reply:
Oh no, please don't